pkg/serverless/proxy/proxy.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build serverlessexperimental
+
+package proxy
+
+import (
+	"io"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// Start starts the proxy
+// This proxy allows us to inspect traffic from/to the AWS Lambda Runtime API
+func Start() {
+	if strings.ToLower(os.Getenv("DD_EXPERIMENTAL_ENABLE_PROXY")) == "true" {
+		log.Debug("the experimental proxy feature is enabled")
+		setup()
+	}
+}
+
+func setup() {
+	originalRuntimeAPIAdress := "127.0.0.1:9001" // todo: fix this hardcoded value
+	reRoutedRuntimeAPIAdress := "127.0.0.1:9000" // todo: fix this hardcoded value
```

Comment:
Nit: reRouted -> rerouted (one word)

Reply:
Also, I would add docs comments to both of these addresses explaining a bit more about what they are.